src/stories/Library/material-search/MaterialSearchHiddenInputs.tsx

```diff
@@ -0,0 +1,42 @@
+interface MaterialSearchHiddenInputsProps {
+  selectedWorkId: string;
+  selectedMaterialType: string;
+}
+
+const MaterialSearchHiddenInputs = ({
+  selectedWorkId,
+  selectedMaterialType,
+}: MaterialSearchHiddenInputsProps) => {
+  // These hidden inputs are used to store the selected work ID and material type in the drupal form
+  // In react, they are located in the storybook component, but in the drupal form they are created in the field widget and hidden with css.
+  return (
+    <div className="material-search__inputs-container">
+      <label className="material-search__label" htmlFor="hidden-work-id-input">
+        Hidden Work ID field
```

Comment:
I see what you mean. 
Since they are not a part of the actual app, i removed it from the material-search component. 

I think it adds value that they are there however, to see props/classes etc, but i moved it into the story instead, so they are still visible in storybook, but not a part of the markup for material-search, similar to what i did in the react repo. 

If you'd rather have it entirely removed let me know.